src/helpers/firebaseHelpers.js

```diff
@@ -10,6 +10,30 @@ export const parseData = (querySnapshot) =>
       {
         id,
         ...product,
+        /***
+         * We most transform the data to have acces
+         * to a propper aria-label and it's color
+         */
+        timeFrameLabel: nextPurchaseDay(
+          product.daysUntilNextPurchase,
+          product.lastPurchaseDate,
+          product.numberOfPurchases,
+        ),
       },
-    ];
+    ].sort((a, b) => {
```

Comment:
Interesting, I have some comments for this line 23:

1. Sometimes is a good idea for "small tasks" to work together. However, in majority of cases, people suggest that you avoid joining "differents" responsibilities in an only function. It's much better to keep simple and isolate functionalities. ["Single-responsibility" principle](https://en.wikipedia.org/wiki/Single-responsibility_principle).
2. Following the first point, I suggested extracting your sorting logic in another function like `sortProductBySoon` or `sortProductByTime`.